10.0/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM debian:jessie
+MAINTAINER Odoo S.A. <info@odoo.com>
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        apt-get update \
+        && apt-get install -y --no-install-recommends \
+            ca-certificates \
+            curl \
+            node-less \
+            node-clean-css \
+            python-pyinotify \
+            python-renderpm \
+            python-support \
+        && curl -o wkhtmltox.deb -SL http://nightly.odoo.com/extra/wkhtmltox-0.12.1.2_linux-jessie-amd64.deb \
+        && echo '40e8b906de658a2221b15e4e8cd82565a47d7ee8 wkhtmltox.deb' | sha1sum -c - \
+        && dpkg --force-depends -i wkhtmltox.deb \
+        && apt-get -y install -f --no-install-recommends \
+        && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false -o APT::AutoRemove::SuggestsImportant=false npm \
+        && rm -rf /var/lib/apt/lists/* wkhtmltox.deb
+
+# Install Odoo
+ENV ODOO_VERSION 10.0
+ENV ODOO_RELEASE 20161007
+RUN set -x; \
+        curl -o odoo.deb -SL http://nightly.odoo.com/${ODOO_VERSION}/nightly/deb/odoo_${ODOO_VERSION}.${ODOO_RELEASE}_all.deb \
+        && echo '1d7a801c9103167bfe7da0ae4191104992e16924 odoo.deb' | sha1sum -c - \
+        && dpkg --force-depends -i odoo.deb \
+        && apt-get update \
+        && apt-get -y install -f --no-install-recommends \
+        && rm -rf /var/lib/apt/lists/* odoo.deb
+
+# Copy entrypoint script and Odoo configuration file
+COPY ./entrypoint.sh /
+COPY ./odoo-server.conf /etc/odoo/
+RUN chown odoo /etc/odoo/odoo-server.conf
+
+# Mount /var/lib/odoo to allow restoring filestore and /mnt/extra-addons for users addons
+RUN mkdir -p /mnt/extra-addons \
+        && chown -R odoo /mnt/extra-addons
+VOLUME ["/var/lib/odoo", "/mnt/extra-addons"]
+
+# Expose Odoo services
+EXPOSE 8069 8071
+
+# Set the default config file
+ENV OPENERP_SERVER /etc/odoo/odoo-server.conf
```

Comment:
Here we should use the freshly updated environment variable `ODOO_RC` instead of the deprecated `OPENERP_SERVER` (which made little sense). See also [the source code](https://github.com/odoo/odoo/blob/e018fd8b2b448cc75a8b6398ba90dc769f742db0/odoo/tools/config.py#L366).